### ISSUE TITLE
Adjust Script, Map and Lists Parameter Type

### ISSUE THREAD

**Issue by pinussilvestrus** [bpmn-io/bpmn-js-properties-panel#354]
__Is your feature request related to a problem? Please describe__

As a follow-up step of #344 we should also adjust the other types to improve the creation of parameters. Cf. following screenshot of the Prototype:

![image](https://user-images.githubusercontent.com/9433996/91159365-46915a00-e6c8-11ea-867e-f20c1cbdd8d0.png)

__Describe the solution you'd like__

* Split up 'Script' to 'Inline Script' and 'External Script Resource' to prevent multiple selections
* Only display List and Map type if specified, but deprecate them (?)

__Describe alternatives you've considered__

Do not deprecate Lists and Maps. Prototyping reveals they can be replaced by expression, but we should be really sure whether we want to remove this.

----

Depends on #344

**Comment by barmac** [comment]
I am closing this as `wontfix` because we released a complete rewrite of the properties panel as v1.0.0. Please feel free to reopen this issue if the problem persists in the 1.x series.
